### ISSUE TITLE
smithay-drm-extras dependency assumes smithay is in parent dir

### ISSUE THREAD

**Issue by rmsthebest** [YaLTeR/niri#1]
Hi,
Cool project, I had never heard about PaperWM before, the sliding thing is really neat.

Trying to clone and build I noticed it failed because of this in the `Cargo.toml`
```toml 
smithay-drm-extras = { version = "0.1.0", path = "../smithay/smithay-drm-extras" }
```

I fixed it by making `smithay-drm-extras` an optional reexport in `smithay` and then updating the use in `tty.rs`. I don't want to intrude on your project, but can send my changes if you are interested.

**Comment by YaLTeR** [comment]
Oh, oops. Locally I have smithay as a path dependency, and I forgot to update this one.

**Comment by YaLTeR** [comment]
Fixed in 0aa5f4c913d1987877beda5a0270b7259bd26d16. Thanks!
